extension/src/commands.ts

```diff
@@ -32,12 +32,63 @@ export function registerCommands(context: vscode.ExtensionContext){
 		}
 		catch (err) {
 			if (err){
-				let msg = localize("autolispext.commands.addFoldingRegion", "Failed to insert snippet");
+				let msg = AutoLispExt.localize("autolispext.commands.addFoldingRegion", "Failed to insert snippet");
 				showErrorMessage(msg, err);
 			}
 		}
 	}));
 
 
+	// Associated with the right click "Generate Documentation" menu item
+	context.subscriptions.push(vscode.commands.registerCommand('autolisp.generateDocumentation', async () => {
+		try {
+			const pos = vscode.window.activeTextEditor.selection.start;
```

Comment:
I suggest that we create a separate .ts file to hold the implementation, and call that implementation here

Reply:
Yes, I have been wanting to do that and this being something of an intermediate PR to break up some larger objective would make it a good one to take care of that.

Reply:
This was not implemented in the latest commit because I am not sure where to offload it to. Either the `commandTasks.ts` I previously suggested or maybe the `/src/help/` since this is kind of augmenting the data related to it?

Reply:
good idea

Reply:
Well that wasn't very clear so here are 2 options:
* Option1: `/src/commandTasks.ts`
* Option2: `/src/help/userDocumentation.ts`

Reply:
I personally like opt. 2, but it's up to you.